contrib/dnsmasq/template.py

```diff
@@ -0,0 +1,32 @@
+pkgname = "dnsmasq"
+pkgver = "2.90"
+pkgrel = 0
+build_style = "makefile"
+make_install_args = [
+    "BINDIR=/usr/bin",
+]
+hostmakedepends = [
+    "pkgconf",
+]
+makedepends = [
+    "dbus-devel",
+    "libnetfilter_conntrack-devel",
+    "linux-headers",
+    "nettle-devel",
+]
+pkgdesc = "DNS, DHCP, RA, TFTP and PXE server"
```

Comment:
since the service will depend on dbus, it should also runtime-depend on dbus here